src/routes/Projects/routes/Project/containers/TreeView/TreeView.js

```diff
@@ -81,10 +74,10 @@ export default class TreeView extends Component {
   }
 
   getStructure = () => {
-    if (!this.props.files || !this.props.files.length) {
+    if (!this.props.files || !size(this.props.files)) {
       return null
     }
-    return this.props.files.map((entry, i) => {
+    return map(this.props.files, (entry, i) => {
```

Comment:
Lodash `map` works for objects. Then the `i` becomes the key of the item within Firebase